src/docs/transactions.dox

```diff
@@ -234,6 +245,16 @@ does not automatically reset the maximum commit timestamp it is tracking. The
 application should explicitly do so by setting a commit timestamp in
 WT_CONNECTION::set_timestamp.
 
+| Timestamp | Description | Constraint |
+|-----------|-------------|------------|
+| all_committed | The oldest timestamp at which all previous write transactions have committed. | |
+| last_checkpoint | The point at which the last checkpoint ran. | last_checkpoint <= stable timestamp |
+| oldest | Modification history is not guaranteed to exist prior to the oldest timestamp, as such reads should not be performed before the oldest timestamp. It can be set via the api. | 0 <= oldest <= stable |
+| oldest_reader | The timestamp of the oldest currently active read transaction, oldest_reader will not exist without an active read transaction. | |
+| pinned | Minimum of the oldest_reader and oldest timestamp. | |
+| recovery | The most recent stable checkpoint prior to shutdown. | |
+| stable | Any transactions before the stable timestamp will not be able to write data, except in the instance of prepared transactions. This timestamp can be set via the api. | stable >= oldest |
```

Comment:
bq. Any transactions before the stable timestamp will not be able to write data, except in the instance of prepared transactions.

Could you clarify what `before` means here? Perhaps `transaction with a commit timestamp older than the stable timestamp`

bq. This timestamp can be set via the api.

The text for this should match that for `oldest` above.